src/index.js

```diff
@@ -1,2 +1,55 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render(count = 0) {
+  function handleClick() {
+    render(count + 1);
+  }
+
+  function handleClickNumber(number) {
+    render(number);
+  }
+  const element = (
+    <div id='hello' className='greeting'>
```

Comment:
HTML에서는 속성에 값을 할당할 때는 쌍따옴표(`"`)을 더 많이 사용하는 것 같아요.
반대로 자바스크립트에서는 `'`을 더 많이 사용하는 것 같아요

```suggestion
    <div id="hello" className="greeting">
```

Reply:
vscode 환경에서 저장을 하면 코드가 자동으로 수정되는데(prettier extension의 영향 때문이라고 생각합니다), 이 수정된 코드를 eslint는 오류라고 하는 문제가 있습니다. npx eslint --fix src/index.js을 통해 자동으로 수정과 저장이 되기 때문에 이번엔 이렇게 해결했지만, eslint와 prettier 충돌에 대해 더 알아보겠습니다. 책에 있는 텍스트와 함께 자세하게 설명해주셔서 감사합니다!